.gitignore

```diff
@@ -159,4 +160,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+.idea/
```

Comment:
I guess I've never asked the team, because I assumed everybody rocked VSCode (or vim if you're a monster).

Reply:
oh yeah... I like PyCharm for python ...